prime-router/settings/staging/0025-WA-Update.yml

```diff
@@ -0,0 +1,95 @@
+- name: "wa-phd"
+  description: "Washington Department of Health"
+  jurisdiction: "STATE"
+  stateCode: "WA"
+  countyName: null
+  senders: []
+  receivers:
+    - name: "elr"
+      organizationName: "wa-phd"
+      topic: "covid-19"
+      translation: !<HL7>
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "WADOHPHRED"
+        receivingApplicationOID: "2.16.840.1.113883.3.237.4.2"
+        receivingFacilityName: "dn1fro00"
+        receivingFacilityOID: "1.3.6.1.4.1.38630.2.1.1.19"
+        messageProfileId: null
+        reportingFacilityName: "7uycso49"
+        reportingFacilityId: "1.3.6.1.4.1.38630.2.1.1.519"
+        reportingFacilityIdType: "ISO"
+        suppressQstForAoe: false
+        suppressHl7Fields: null
+        suppressAoe: false
+        defaultAoeToUnknown: false
+        useBlankInsteadOfUnknown: null
+        truncateHDNamespaceIds: false
+        usePid14ForPatientEmail: false
+        convertTimestampToDateTime: null
+        processingModeCode: null
+        nameFormat: "washington"
+        receivingOrganization: null
+        type: "HL7"
+      jurisdictionalFilter:
+        - "matches(abnormal_flag, A)"
+        - "orEquals(ordering_facility_state, WA, patient_state, WA)"
+      qualityFilter: []
+      reverseTheQualityFilter: false
+      deidentify: false
+      timing:
+        operation: "MERGE"
+        numberPerDay: 720
+        initialTime: "00:00"
+        timeZone: "EASTERN"
+        maxReportCount: 500
+      description: ""
+      transport: !<AS2>
+        receiverUrl: "https://uat-onehealthport-as2.axwaycloud.com/exchange/ZZOHPUAT"
+        receiverId: ZZOHPUAT
+        senderId: 7uycso49
+        type: "AS2"
```

Comment:
Is this intentional?   it looks like we're sending to their production service, from Staging.